integration-tests/src/test/java/gov/uk/address/api/stepdefinitions/AddressSteps.java

```diff
@@ -233,10 +233,10 @@ public void theUserArrivesAtFindYourAddressPage() throws IOException, Interrupte
                         this.testContext.getSessionId()));
     }
 
-    @When("user requests lands on \\/addresses\\/v2")
-    public void userRequestsLandsOnAddressesV2() throws IOException, InterruptedException {
+    @When("user requests lands on \\/addresses")
```

Comment:
Are there v1 tests to remove or no?

Reply:
There were none :D - Ola had created tests for V2